WORKSPACE

```diff
@@ -1,5 +1,11 @@
 workspace(name = "envoy")
 
+git_repository(
+    name = "io_bazel_rules_go",
+    remote = "https://github.com/bazelbuild/rules_go.git",
+    commit = "4374be38e9a75ff5957c3922adb155d32086fe14",
+)
+
```

Comment:
Let me test, but I don't think you need to add it here, since it's going to be pulled by `envoy_dependencies()` anyway.